utils/bash-completion/POST_BUILD

```diff
@@ -0,0 +1,2 @@
+# relin util-linux because some completions are overlaping with bash-completion
+lin -c util-linux
```

Comment:
@Florin65 this is not a good idea, we should resolve the conflict. Won't merge this.